setup.py

```diff
@@ -54,7 +54,7 @@
 third_party_path = os.path.join(base_dir, 'third_party')
 
 _libtpu_version = '0.1.dev20210615'
```

Comment:
Can you also update this link to `dev20211015 `? Same for the 1.10 since `dev20211013` wheel has some issue.

Reply:
done